electricpy/__init__.py

```diff
@@ -5943,157 +6220,164 @@ def vipf(V=None,I=None,PF=1,find=''):
-    
+
     Given the angular velocity in rad/sec, this
     function will evaluate the velocity in RPM
     (Revolutions-Per-Minute).
-    
+
     Parameters
     ----------
     rad:        float
                 The angular velocity in radians-
                 per-second
-    
+
     Returns
     -------
     rpm:        float
                 The angular velocity in revolutions-
                 per-minute (RPM)
     """
-    rpm = 60/(2*_np.pi)*rad
-    return(rpm)
+    rpm = 60 / (2 * _np.pi) * rad
+    return (rpm)
+
 
 # Define Angular Velocity Conversion Functions
 def rpm_to_rad(rpm):
     """
     RPM to Radians-per-Second Converter
-    
+
     Given the angular velocity in RPM (Revolutions-
     Per-Minute), this function will evaluate the
     velocity in rad/sec.
-    
+
     Parameters
     ----------
     rpm:        float
                 The angular velocity in revolutions-
                 per-minute (RPM)
-    
+
     Returns
     -------
     rad:        float
                 The angular velocity in radians-
                 per-second
     """
-    rad = 2*_np.pi/60*rpm
-    return(rad)
+    rad = 2 * _np.pi / 60 * rpm
+    return (rad)
+
 
 # Define Angular Velocity Conversion Functions
 def hz_to_rpm(hz):
     """
     Hertz to RPM Converter
-    
+
     Given the angular velocity in Hertz, this
     function will evaluate the velocity in RPM
     (Revolutions-Per-Minute).
-    
+
     Parameters
     ----------
     hz:         float
                 The angular velocity in Hertz
-    
+
     Returns
     -------
     rpm:        float
                 The angular velocity in revolutions-
                 per-minute (RPM)
     """
-    rpm = hz*60
-    return(rpm)
+    rpm = hz * 60
+    return (rpm)
+
 
 # Define Angular Velocity Conversion Functions
 def rpm_to_hz(rpm):
     """
     RPM to Hertz Converter
-    
+
     Given the angular velocity in RPM (Revolutions-
     Per-Minute), this function will evaluate the
     velocity in Hertz.
-    
+
     Parameters
     ----------
     rpm:        float
                 The angular velocity in revolutions-
                 per-minute (RPM)
-    
+
     Returns
     -------
     hz:         float
                 The angular velocity in Hertz
     """
-    hz = rpm/60
-    return(hz)
+    hz = rpm / 60
+    return (hz)
+
 
 # Define Synchronous Speed Calculator
-def syncspeed(Npol,freq=60,Hz=False):
+def syncspeed(Npol, freq=60, Hz=False):
     """
     Synchronous Speed Calculator Function
-    
+
     Simple method of calculating the synchronous
     speed of an induction machine given the number
     of poles in the machine's construction, and
     the machine's operating electrical frequency.
-    
+
     .. math:: \\omega_{\\text{syn}}=\\frac{2\\pi
        \\cdot\\text{freq}}{\\frac{N_{\\text{pol}}}{2}}
-    
+
     Parameters
     ----------
     Npol:       int
-                Number of electrical poles in
+                Number of electric
+al poles in
```

Comment:
this looks like a little over-zealous refactoring. We'll need to get this brought back onto one line.

Reply:
this was fixed in updated PR